packages/context-menu/src/styles/vaadin-menu-overlay-styles.js

```diff
@@ -5,7 +5,7 @@
  */
 import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-export const styles = css`
+export const menuOverlayStyles = css`
```

Comment:
Could also update this file to import from `lit`.

Reply:
I'm planning to cover that in menu-bar's own PR.